### PR TITLE
feathr Kafka streaming docs

### PR DESCRIPTION
**Pull request by daxsorbito**
This PR includes, kafka producer file and a notebook that uses `feathr` to ingest the produced topics. See README.md for more details.

### CODE REVIEW

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (ivaramme) [inline feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: Can this URL be moved to a variable?

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (ivaramme) [inline feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: This too and to its effect any hardcoded URLs

File: feathr-example/streaming-source-ingestion/eventhub_producer.py
Comment (ivaramme) [inline feathr-example/streaming-source-ingestion/eventhub_producer.py]: 👍🏽

**Review by ivaramme — COMMENTED** [review]
Looks good to me in its majority. Can you take a look at my comments, if it isn't possible, I'll go ahead and approve, but if it is possible to make the change, I would rather making it or at 
least removing the strings pointing to our Azure resources.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (nightism) [inline feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: NIT: typo KAFKA_SASL_**JAAS**_CONFIG

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (nightism) [inline feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: NIT: typo -> true

File: feathr-example/streaming-source-ingestion/eventhub_producer.py
Comment (nightism) [inline feathr-example/streaming-source-ingestion/eventhub_producer.py]: Just curious what the purpose is of saving this to environment variable? I think the changes will be discarded after the script finishes?

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (nightism) [inline feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb:312]: Nit: I would suggest to add a link to the Feathr documentation for this part to explain how the streaming data is loaded - i.e. whether is a long running job or it will materialise for only a period of time. Or we can add some comments here to explain what those `timeout` parameters mean.

**Review by nightism — CHANGES_REQUESTED** [review]

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: Moved to a variable.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: Removed the databricks configuration as it is not needed in this example.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: Thanks, fixed this.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb]: Thanks, fixed this.

File: feathr-example/streaming-source-ingestion/eventhub_producer.py
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/eventhub_producer.py]: The previous implementation was using the environment util class, right now it is just passed directly as part of the config. Would be removing this.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb:312]: Based on the setting, it is a long-running job that the user would set through the `streamingTimeoutMs`.
1. `streamingTimeoutMS` parameter of `RedisSink` is the value to set on how long the spark job would ingest the kafka topics.
1. `timeout_sec` parameter of `wait_job_to_finish` method is the value on how long would the feathr client to wait until when there is no response from the spark job.

File: feathr-example/streaming-source-ingestion/eventhub_producer.py
Comment (daxsorbito) [reply feathr-example/streaming-source-ingestion/eventhub_producer.py]: Removed this code as it is not needed.

File: feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb
Comment (nightism) [reply feathr-example/streaming-source-ingestion/streaming_ingestion.ipynb:312]: sure thanks 🙏

**Review by nightism — APPROVED** [review]

**Pull request merged**
